pkg/server/server.go

```diff
@@ -669,6 +669,18 @@ func (s *Server) HTTPHandler() http.Handler {
 		}
 	}))
 
+	handle("/api/helm/release/async", authHandlerWithUser(func(user *auth.User, w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case http.MethodPost:
+			helmHandlers.HandleHelmInstallAsync(user, w, r)
+		case http.MethodPut:
+			helmHandlers.HandleUpgradeReleaseAsync(user, w, r)
+		default:
+			w.Header().Set("Allow", "GET, POST, PATCH, PUT, DELETE")
```

Comment:
At the moment this only supports POST and PUT

```suggestion
			w.Header().Set("Allow", "POST, PUT")
```